package.json

```diff
@@ -125,6 +125,7 @@
     "react-router": "^2.3.0",
     "react-test-renderer": "^15.3.2",
     "redux": "^3.0.0",
+    "sitemap": "~1.8.2",
```

Comment:
Can you use ^ instead of ~ like other package?
Also can you use yarn so the lock file is updated?